modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc

```diff
@@ -2,13 +2,23 @@
 :description: Use converters to handle the serialization and deserialization of data between a Redpanda topic and a managed connector.
 :page-cloud: true
 
-Converters handle the serialization and deserialization of data between a Redpanda topic and a connector. 
+Connectors are a translation layer working between the Redpanda and the remote system. 
+
+For *sink* connectors the translation happens in the following phases:
+
+. Converter deserializes data from Redpanda message format (for example JSON or Avro) to a universal in-memory connect data format.
+. The in-memory Connect data structure is translated by the connector to the data model of the remote system.
+
+For *source* connectors it is vice versa, the phases are:
+
+. Connector translates the data model from remote system format to the in-memory Connect data structure.
+. Converter serializes the data from a universal in-memory Connect format to a Redpanda message.
 
 Each Redpanda message is a key and value record. Record key and value converters are configured separately with the `Redpanda message key format` and `Redpanda message value format` properties. Key and value converters can be different.
 
 [NOTE]
 ====
-If an external system requires structured data (like BigQuery or a SQL database), then you must provide data with a schema. Use the Avro, Protobuf, or JSON converter with a schema.
+If an external system requires structured data (like BigQuery or a SQL database), then you must provide data with a schema. Use the Avro, Protobuf or JSON converter with a schema.
```

Comment:
This is the Oxford comma, which we adhere to.
https://github.com/redpanda-data/docs-site/blob/main/meta-docs/STYLE-GUIDE.adoc#commas